web/public/js/directives/cd-dojos-map.js

```diff
@@ -58,6 +59,14 @@ function cdDojosMap($stateParams, cdDojoService, $state) {
             var path = urlSlugArray.join('/');
             $state.go('dojo-detail', {country:country, path:path });
           }
+
+          $scope.getDojoURL = function(dojo) {
```

Comment:
You should inject dojoUtils in this file too and use it here instead of writing a function just for this file.